xdn-magento-pwa-studio-example/deploying-Magento-PWA-studio-on-XDN.md

```diff
@@ -0,0 +1,309 @@
+# Deploying Magento PWA studio on XDN
+
+## 1. Install Magento PWA Studio
+
+- Scaffold Magento PWA Studio demo: `npm init @magento/pwa` (more info: https://magento.github.io/pwa-studio/tutorials/pwa-studio-fundamentals/project-setup/)
+- Navigate to the project folder: `cd %project_name%`
+- Build the app `dist` folder: `npm run build`
+
+## 2. Install XDN
+
+- Install XDN packages: `npm i -D @xdn/cli@latest && npm i @xdn/core@latest @xdn/prefetch@latest @xdn/devtools@latest`
+- Next steps will cover the XDN key files you need to add. For more information about them see https://developer.moovweb.com/guides/starter
+- Add `xdn.config.js` file with your origin host. For example:
+```js
+module.exports = {
+  routes: './xdn/routes.js',
+  backends: {
+    origin: {
+      domainOrIp: 'master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud',
+      hostHeader: 'master-7rqtwti-mfwmkrjfqvbjk.us-4.magentosite.cloud',
+    },
+  },
+};
+
+```
+- Add `xdn/service-worker.js` file:
```

Comment:
Can we re-word this to suggest first running `xdn init`? This will do the install of the packages automatically, creating the config file, etc. Then add the notes to update the backend.